x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -355,9 +368,9 @@ export const buildEqlDataProviderOrFilter = (
       ),
     };
   } else if (!Array.isArray(ecs)) {
-    const alertGroupIdField: string[] = getField(ecs, ALERT_GROUP_ID);
+    const alertGroupIdField = getField(ecs, ALERT_GROUP_ID);
     const queryMatchField = getFieldKey(ecs, ALERT_GROUP_ID);
-    const alertGroupId = alertGroupIdField?.length ? alertGroupIdField[0] : 'unknown-group-id';
+    const alertGroupId = alertGroupIdField?.length ? alertGroupIdField : 'unknown-group-id';
```

Comment:
If `alertGroupIdField` is not array any more, why we have this length check `alertGroupIdField?.length`?